macosx/Info.plist.in

```diff
@@ -61,7 +61,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>14714.@TR_VERSION_MAJOR@.@TR_VERSION_MINOR@.@TR_VERSION_PATCH@</string>
+	<string>14714.@TR_VERSION_MAJOR@.@TR_VERSION_MINOR@.@TR_VERSION_PATCH@.@TR_VERSION_BETA_NUMBER@</string>
```

Comment:
`@TR_VERSION_BETA_NUMBER@` is [sometimes an empty string](https://github.com/transmission/transmission/blob/2a57b17031053999e66c7c094e5d6cd6ec4666aa/CMakeLists.txt#L85) when the release isn't a beta.

Is it OK for this string to end in a `.`?

Reply:
Yes it's fine, I tested this scenario; `@"".integerValue` is 0.

Reply:
I suppose the concern here is that [`CFBundleVersion` docs](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion) describes the value as "a machine-readable string composed of one to three period-separated integers" that "can include more integers". So `1.2.3.` (ending in a period [i.e. separator] that's not followed by a numberic component [that it's meant to separate]) doesn't fit the description. I.e. it's not about whether an empty string could be successfully parsed as an integer.

Reply:
Is there a way to drop the period when the end is missing?

Reply:
Marking as resolved since the PR no longer appends `.@TR_VERSION_BETA@`